### ISSUE TITLE
TestFlight automatic update on devices creates "Play [BU] has expired" alert 

### ISSUE THREAD

**Issue by pyby** [SRGSSR/playsrg-apple#261]
### Issue type

Incorrect behavior

### Description of the problem

Since weeks, some users reported that latest TestFlight build installed on their devices displays “ Play [BU] has expired”, even if the TestFlight apps tell that the build will expired in 90 days.

Manually reinstalling the build from TestFlight app, the Play app can be launch as expected.
Tested on iOS 16.1.2 and iPadOS 15.7.1

Logs from iOS console:
```
Springboard (FrontBoardServices):
[APP_BUNDLE_ID - signature state: Profile Missing, reason: Provisioning Profile Not Found
Springboard (Springboard):
Attempted to launch an untrusted application scene sceneID:APP_BUNDLE_ID-default
Springboard (UIKitCore):
_willShowAlertController: <_SBAlertController: 0xdd0781200; title: "“APP_DISPLAY_NAME” Beta Has Expired">
```

### Environment information

* Application version: Any
* iOS / tvOS version: Any
* Device: Any

### Reproducibility

Difficult to reproduce

### Steps to reproduce

1. Building on CI. Code signing the app with cloud signing.
2. Upload it AppStoreConnect et distribute it.
3. iPhone and Apple TV devices have the TestFlight apps with "automatic update" enabled.
4. The build is installed in background on devices.
5. try to open the application from the SpringBoard.

**Comment by pyby** [comment]
https://developer.apple.com/forums/thread/720033
http://openradar.appspot.com/radar?id=5536973938753536

Workaround:
1. Open TestFlight application.
2. Select the _Play app_ to open the detail page.
3. Select "Previous build".
4. Install manually a previous build. (It will disable automatic update).
5. Install manually the latest build.

The application can be open.

6. Optionally, reenable the "automatic update" feature from the _Play app_ detail page in TestFlight application.

**Comment by pyby** [comment]
I closed the issue on Apple feedbacks: FB11864073 (on open radar as well)
it seems to be fixed server side.
https://developer.apple.com/forums/thread/720033?page=4
